pkg/solana/config_digester.go

```diff
@@ -57,6 +57,7 @@ func (d OffchainConfigDigester) ConfigDigest(cfg types.ContractConfig) (types.Co
 		return digest, err
 	}
 
+	cfg.OnchainConfig = []byte{} // TODO: on chain config digest does not implement onchainConfig
```

Comment:
What is the objective of this TODO? What is exactly missing here?

Reply:
updated to a comment without the TODO (there's not todo, just a comment pointing back to the program)